LogTarget.php

```diff
@@ -50,11 +50,18 @@ public function export()
         $summary = $this->collectSummary();
         $dataFile = "$path/{$this->tag}.data";
         $data = [];
+        $exceptions = [];
         foreach ($this->module->panels as $id => $panel) {
-            $data[$id] = $panel->save();
+            try {
+                $data[$id] = serialize($panel->save());
```

Comment:
I think the reasoning is that we can catch individual serialize errors here so that other panels continue to function.